cccs-build/superset/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Vault CA container import
 ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_11376_a25c34e1
 FROM $VAULT_CA_CONTAINER AS vault_ca
-FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.0_20230315183034_b6498
+FROM uchimera.azurecr.io/cccs/superset-base:feature_CLDN-2076_20230414180632_b6607
```

Comment:
we should change this back to a cccs branch

Reply:
Right, and I don't plan to merge it in cccs-2.0 even if it's what the PR says, I'll wait for cccs-2.1.